packages/nc-gui/components/workspace/EmptyPlaceholder.vue

```diff
@@ -18,22 +18,18 @@ const openCreateProjectDlg = (type: NcProjectType) => {
   <div class="flex items-center justify-center mt-8">
     <div class="flex flex-col gap-4 items-center text-gray-500">
       <NcIconsInbox />
-      <div class="font-weight-medium">No Projects</div>
-      <template v-if="props.buttons">
-        <div class="text-xs">Create Base</div>
-        <div class="flex gap-2 justify mt-1">
-          <a-button class="flex-1 nc-btn" @click="openCreateProjectDlg(NcProjectType.DB)">
-            <div class="flex gap-2 items-center justify-center text-xs">
-              <GeneralProjectIcon :type="NcProjectType.DB" class="text-[#2824FB] text-lg" />
-              New Database Base
-            </div>
-          </a-button>
-        </div>
-
-        <WorkspaceCreateProjectDlg v-model="baseCreateDlg" :type="baseType" />
-      </template>
+      <div class="font-medium text-gray-500">No Projects in workspace</div>
+      <div class="flex gap-2 justify mt-1">
```

Comment:
if `props.buttons` is removed, then `props` is never used. Hence you can remove it.